### PR TITLE
Add locked letters feature

### PR DESCRIPTION
**Pull request by rasmusroessum**
This commit is adding a feature. This feature will give the user a onClick feature, that will lock the letter.

This means, that a user can now try all kinds of words without keeping it all in their thoughts.

I think this should be tested, but it seems to work. I made some comments in the code also, for the future.

I can also see, that it's like I got some older commit in - but I'm not sure..

### CODE REVIEW

**Comment by joglr** [comment]
#### 📝 Changed routes:


| [**/**](https://ordle-88r0wipr3-joglr.vercel.app/) |
|:-:|
|<a href="https://ordle-88r0wipr3-joglr.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/U5t7iHxi2zNa60XbN-G_5N6dmdu6hvWBufRowH9Ut1r9bcpUf91jH8kvpKPraeWw_AfTCpbJh2SaOaL9iL2r0w.png">(view full size)</a> |







Commit 8c38121c7ab4a8290c9ba4a6506720dcc42caf98 (https://ordle-88r0wipr3-joglr.vercel.app).

File: src/state.ts [hunk 1]
```diff
@@ -49,7 +49,7 @@ export type History = HistoryEntry[];
 
 export type BoardState = {
   history: History;
-  currentAttempt: string[] | null[];
+  currentAttempt: string[] | null[] | any[];
```
Comment (joglr) [inline src/state.ts:52]: I would like to avoid any's in the codebase :(

Comment (rasmusroessum) [reply src/state.ts:52]: Yes - I know, but I don't know how to do it...

File: src/styles/Home.module.css [hunk 2]
```diff
@@ -44,6 +44,15 @@
   animation: jump 100ms ease-in-out;
   animation-fill-mode: forwards;
 }
+/* Do not use important ... */
```
Comment (joglr) [inline src/styles/Home.module.css:47]: Do not use important

**Review by joglr — APPROVED** [review]
Nice work!

**Pull request merged**